<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>606</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>606</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>606</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>XP3Player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/PC/Downloads/Без имени-1.png</normaloff>C:/Users/PC/Downloads/Без имени-1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
color:
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color:rgb(231, 229, 255);
}</string>
   </property>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>440</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(201, 199, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}
QPushButton::pressed{
    background-color: rgb(149, 130, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Add new track</string>
    </property>
   </widget>
   <widget class="QListWidget" name="track_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView {
    show-decoration-selected: 1; /* make the selection span the entire width of the view */
}

QListView::item:alternate {
    background: #EEEEEE;
}

QListView::item:selected {
    border: 1px solid #6a6ea9;
}

QListView::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ABAFE5, stop: 1 #8588B2);
}

QListView::item:selected:active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #6a6ea9, stop: 1 #888dd9);
}

QListView::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #FAFBFE, stop: 1 #DCDEF1);
}</string>
    </property>
    <property name="autoScrollMargin">
     <number>20</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="time_line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>741</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSlider::groove:horizontal {
     border: 1px solid #999999;
     height: 8px; /* по умолчанию бороздка расширяется до размеров ползунка. задав высоту она принимает фиксированный размер */
     background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
     margin: 2px 0;
 }

 QSlider::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 1px solid #5c5c5c;
     width: 18px;
     margin: -2px 0; /* рукоятка располагается по умолчанию в прямоугольнике содержимого бороздки. Расширяется наружу от бороздки */
     border-radius: 3px;
 }</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="track_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background-color: rgb(201, 199, 255);
	font: bold 14px;
 }</string>
    </property>
    <property name="text">
     <string>Playing</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSlider::groove:horizontal {
     border: 1px solid #999999;
     height: 8px; /* по умолчанию бороздка расширяется до размеров ползунка. задав высоту она принимает фиксированный размер */
     background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
     margin: 2px 0;
 }

 QSlider::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 1px solid #5c5c5c;
     width: 18px;
     margin: -2px 0; /* рукоятка располагается по умолчанию в прямоугольнике содержимого бороздки. Расширяется наружу от бороздки */
     border-radius: 3px;
 }</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>248</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QPushButton" name="next_button">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(201, 199, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}
QPushButton::pressed{
    background-color: rgb(149, 130, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}
</string>
       </property>
       <property name="text">
        <string>next</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="prev_button">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(201, 199, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}
QPushButton::pressed{
    background-color: rgb(149, 130, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}</string>
       </property>
       <property name="text">
        <string>prev</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="stop_button">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(201, 199, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}
QPushButton::pressed{
    background-color: rgb(149, 130, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}</string>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="play_button">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(201, 199, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}
QPushButton::pressed{
    background-color: rgb(149, 130, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}</string>
       </property>
       <property name="text">
        <string>play</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="del_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(201, 199, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}
QPushButton::pressed{
    background-color: rgb(149, 130, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: grey;
    font: bold 14px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>delete current track</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>16</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
